rsrcdump/structtemplate.py

```diff
@@ -118,8 +123,8 @@ def pack(self, obj: Any) -> bytes:
                 buf += self.pack_record(item)
             return buf
 
-    def pack_record(self, json_obj: Any) -> bytes:
-        def process_json_field(_field_format, _field_value):
+    def pack_record(self, json_obj: list[str] | dict[str, str] | str) -> bytes:
+        def process_json_field(_field_format: str, _field_value: str | bytes) -> str | bytes:
```

Comment:
`_field_value` can be any JSON scalar. It can also be an int, a float, a bool, etc.

Reply:
Updated to be more accurate in e907f63